vhdbuilder/packer/install-dependencies.sh

```diff
@@ -105,6 +105,7 @@ cat << EOF >> ${VHD_LOGS_FILEPATH}
 EOF
 
 VNET_CNI_VERSIONS="
+1.2.7
 1.2.6
 1.2.0_hotfix
 1.2.0
```

Comment:
Can remove this

Reply:
Also update SWIFT_CNI_VERSIONS below? once the versions converge I'll combine them.

Reply:
@paulgmiller Do you mean 1.2.0 or 1.2.6 or both?

Reply:
Removed Linux cni 1.2.0 and added swift cni 1.2.7

Reply:
Thanks that's what I meant